### ISSUE TITLE
FontStyle and FontSize

### ISSUE THREAD

**Issue by Aditmimin** [srehanuddin/Cordova-Plugin-Bluetooth-Printer#44]
Hi srehanuddin, thanks for the awesome plugin! btw, can i change font style (bold, italic,etc) or fontsize?

regards,

**Comment by srehanuddin** [comment]
yes you can, i do this with a trick, i made an canvas image with custom fonts and barcode and print image.

**Comment by mxntiong** [comment]
Hi, can you provide a sample on how to print the canvas?

**Comment by sikaili99** [comment]
Hey try these commands if you still need this [Escpos-commands-js](https://github.com/humbertopiaia/escpos-commands-js)
